collection-scripts/gather_nodes

```diff
@@ -87,7 +87,12 @@ do
     oc exec -it $pod -n node-gather -- dmesg >> $NODE_PATH/dmesg
     oc exec -it $pod -n node-gather -- cat /host/proc/cmdline >> $NODE_PATH/proc_cmdline
     oc exec -it $pod -n node-gather -- lspci -vv >> $NODE_PATH/lspci
-    oc exec -it $pod -n node-gather -- cat /host/etc/pcidp/config.json >> $NODE_PATH/pcidp_config.json
+
+    oc exec -it $pod -n node-gather -- [ -f /host/etc/pcidp/config.json ]
+    if [[ $? -eq 0 ]]; then
+        oc exec -it $pod -n node-gather -- cat /host/etc/pcidp/config.json >> $NODE_PATH/pcidp_config.json
```

Comment:
why not use `oc cp` instead ?

Reply:
This comment applies to quite a few items.
If you don't mind, let's just add the condition in this patch, and I'll prepare a separate patch switching "cat ..." -> oc cp

Reply:
Fixed anyway.
Other places are not this trivial (files are not just copied)